connexion/swagger.py

```diff
@@ -0,0 +1,197 @@
+"""
```

Comment:
you can remove the license header as the `LICENSE` file is enough (and we should probably remove all existing file headers in a later PR)